src/org/labkey/test/components/experiment/NodeDetailGroup.java

```diff
@@ -0,0 +1,121 @@
+package org.labkey.test.components.experiment;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class NodeDetailGroup extends WebDriverComponent<NodeDetailGroup.ElementCache>
+{
+    final WebElement _el;
+    final WebDriver _driver;
+    final String _groupName;
+
+    public NodeDetailGroup(WebElement element, String groupName, WebDriver driver)
+    {
+        _el = element;
+        _driver = driver;
+        _groupName = groupName;
+    }
+
+    public String getGroupName()
+    {
+        if (_groupName!=null)
+            return _groupName;
+        else
+            return elementCache().summary.getText();
+    }
+
+    public NodeDetail getItem(String itemName)
+    {
+        return  elementCache().items().stream().filter(a-> a.getName().equals(itemName))
+                .findFirst().orElseThrow();
+    }
+
```

Comment:
I'm still unclear what the difference is between these two functions (getItem & getItemByTitle). They both take a string value and return a NodeDetail object. What is the difference between an nodes name and its title?

Reply:
![image](https://user-images.githubusercontent.com/16809856/80020195-7962e180-848d-11ea-8898-d1c4231a3c14.png)
Title is an attribute on the li element of the NodeDetail, Name is the contents of the link/span that are shown in the body.  Title has a prefix like "Data: " + Name.  This prefix seems to be related to the grouping (like in the image).
We can take away name and force callers to go in via title if that's preferable?  I've been supporting both ways with the thought that it might be easier to round-trip some data if it comes out as name but not title